sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts

```diff
@@ -103,13 +109,51 @@ export class ConfirmationOfEnrollmentService {
     if (!maxTuitionRemittanceData) {
       return null;
     }
+
+    const previousTuitionRemittance = await this.getPreviousTuitionRemittance(
+      maxTuitionRemittanceData.studentAssessment.id,
+      disbursementId,
+    );
+
     return this.getMaxTuitionRemittance(
       maxTuitionRemittanceData.disbursementValues,
       maxTuitionRemittanceData.studentAssessment.offering,
       MaxTuitionRemittanceTypes.Estimated,
+      previousTuitionRemittance,
     );
   }
 
+  /**
+   * Gets the previous disbursement schedule remittance value.
+   * Returns tuition remittance effective amount if present or tuition remittance requested amount if present, otherwise 0.
+   * @param studentAssessmentId for the disbursement schedule.
+   * @param disbursementId disbursement id.
+   * @returns previous disbursement schedule remittance value.
+   */
+  async getPreviousTuitionRemittance(
+    studentAssessmentId: number,
+    disbursementId: number,
+  ): Promise<number> {
+    const previousTuitionRemittanceData =
+      await this.disbursementScheduleRepo.findOne({
+        select: {
+          id: true,
+          tuitionRemittanceEffectiveAmount: true,
+          tuitionRemittanceRequestedAmount: true,
+        },
+        where: {
+          id: LessThan(disbursementId),
```

Comment:
It would be more accurate to rely on the `disbursementDate` instead of the `id`.
The disbursements are saved to DB in the same operation and, even being likely to be ordered by ID, the date would be guaranteed. Does it make sense?

Reply:
Yes, that makes sense. Changed that and adapted the E2E tests. Thanks!